src/Text/Pretty/Simple.hs

```diff
@@ -39,29 +39,42 @@ module Text.Pretty.Simple
   (
   -- * Output with color on dark background
     pPrint
+  , pHPrint
+  , pPrintForceColor
+  , pHPrintForceColor
   , pShow
   , pString
   -- * Aliases for output with color on dark background
   , pPrintDarkBg
+  , pHPrintDarkBg
+  , pPrintForceColorDarkBg
+  , pHPrintForceColorDarkBg
   , pShowDarkBg
   , pStringDarkBg
   -- * Output with color on light background
   , pPrintLightBg
+  , pHPrintLightBg
+  , pPrintForceColorLightBg
+  , pHPrintForceColorLightBg
   , pShowLightBg
   , pStringLightBg
   -- * Output with NO color
   , pPrintNoColor
+  , pHPrintNoColor
   , pShowNoColor
   , pStringNoColor
   -- * Output With 'OutputOptions'
   , pPrintOpt
+  , pHPrintOpt
   , pShowOpt
   , pStringOpt
   -- * 'OutputOptions'
   , OutputOptions(..)
   , defaultOutputOptionsDarkBg
   , defaultOutputOptionsLightBg
   , defaultOutputOptionsNoColor
+  , CheckColorTty(..)
+  , hCheckTTY
```

Comment:
Do we still need to export `hCheckTTY` after your refactor? Probably not, I will remove it.

Reply:
This will also mean that there is no way to write to a string depending on whether `stdout` is a TTY or not, but I don't think it's a problem, since one only has to use a single function from `base` and it's more elegant than the current API anyway.

Reply:
Yeah, I was on the fence about that one.  I'm not sure if it would be useful to end users.  I was going to export it because you had added it, but if you want to take it out, then please go ahead.  People can always get it from the `Internal` module if they really want it.